spec/statement.dd

```diff
@@ -496,8 +496,8 @@ $(P
         will have $(D scope) semantics.
 )
 $(P
-        The aggregate must be loop invariant, meaning that
-        elements to the aggregate cannot be added or removed from it
+        The loop must be $(RELATIVE_LINK2 foreach_restrictions, invariant), meaning that
```

Comment:
I don't like this sentence. The "loop" being invariant as opposed to something being loop invariant is not a clear concept.

Reply:
OK, I've restored the original wording.